docs/auth-rate-limit.md

```
@@ -0,0 +1,49 @@
+# Auth Rate Limit Policy
+
+## 목적
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)